docs/UserGuide.md

```diff
@@ -616,7 +616,8 @@ So if you use `sort n/asc s/asc so/haha`, the required criteria are name and sta
 
 | Term                               | Description                                                                                                                                                                          |
 |------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **AND search**                     | AND search finds all of the keywords.  For example, `find n/Shutter Island` returns only results that contain Shutter and Island.                                                    |
+| **AND search**                     | AND search finds all of the keywords.  For example, `find n/Shutter Island` returns only results that contain Shutter and Island.                |
+| **ASCII**                   | A 8-bit character set containing 128 characters. This includes number 0-9, upper and lower case english Letters (A-Z) and certain special characters.                |
```

Comment:
Should this be `An 8-bit character set` instead?
Also, `This includes the numbers 0-9` instead?
Also, `English letters (A-Z), and` instead?

Reply:
Also should we shift ASCII's glossary entry to the bottom of the table? I don't think we should keep it in the middle of the search entries. Kind of breaks the flow if readers want to compare the 2 searches. Just my 2 cents!